src/main/scala/com/wavesplatform/matcher/MatcherSettings.scala

```diff
@@ -18,7 +20,10 @@ case class MatcherSettings(enable: Boolean,
                            snapshotsInterval: FiniteDuration,
                            maxOpenOrders: Int,
                            priceAssets: Seq[String],
-                           predefinedPairs: Seq[AssetPair]
+                           predefinedPairs: Seq[AssetPair],
+                           maxTimestampDiff: FiniteDuration,
+                           orderHistoryFile: String,
+                           isMigrateToNewOrderHistoryStorage: Boolean
```

Comment:
It's migration from the previous storage in the snapshot to MVStore. It should be done once. Or can be done manually if we need to rebuild history. The settings is enough here